src/Spectre.libStatistics.Tests/MathTest.cpp

```diff
@@ -218,4 +218,11 @@ TEST(AbsTest, returns_absolute_value_of_each_element)
     const auto absolutes = abs(gsl::as_span(data));
     EXPECT_THAT(absolutes, ContainerEq(Data{ 1,2,3,4 }));
 }
+
+TEST(DiffTest, calculates_differential)
```

Comment:
Would be nice to add tests when an empty vector is passed, or when the length of vector is lower than the requested order.

Reply:
Done